app/coffee/Features/Notifications/NotificationsHandler.coffee

```diff
@@ -29,14 +29,16 @@ module.exports =
 					unreadNotifications = []
 				callback(null, unreadNotifications)
 
-	createNotification: (user_id, key, templateKey, messageOpts, expiryDateTime, callback)->
+	createNotification: (user_id, key, templateKey, messageOpts, expiryDateTime, forceCreate, callback)->
 		payload = {
 				key:key
 				messageOpts:messageOpts
 				templateKey:templateKey
 		}
 		if expiryDateTime?
 			payload.expires = expiryDateTime
+		if forceCreate
+			payload.forceCreate = true
```

Comment:
I think I'll add a tech-debt ticket to refactor this interface/service to make `forceCreate` the default behaviour, removing the flag entirely.